js/libs/core.js

```diff
@@ -6,6 +6,9 @@ Fliplet.FormBuilder = (function() {
   var DATE_FORMAT = 'YYYY-MM-DD';
   var LOCAL_FORMAT_DATE = moment.localeData().longDateFormat('L');
 
+  Fliplet.Env.set('ie11', navigator.userAgent.indexOf('Trident/') !== -1);
```

Comment:
Please don't set environment variables on `Env`, that's just for first-party use cases provided by our core team.

I'd recommend using an instance variable you can pass between the two files, @tonytlwu do you agree?

Reply:
Can you please explain how to implement this in the forms builder, because I do not understand how the logic is arranged in this component and how to set the instance variable

Reply:
If you set up an instance variable in the Form Vue instance doesn't that then become available to the child components instances when accessing the parent?

Reply:
I find a solution, fixed